src/applications/disability-benefits/all-claims/tests/cypress.helpers.js

```diff
@@ -186,50 +173,14 @@ export const setup = (cy, toggles = mockFeatureToggles) => {
         servicePeriods: data.serviceInformation.servicePeriods,
         reservesNationalGuardService:
           data.serviceInformation.reservesNationalGuardService,
+        includeToxicExposure: enableContent,
       },
       metadata: mockPrefill.metadata,
     });
   });
 };
 
-/**
- * Build a list of unreleased pages using the given toggles
- *
- * @param {object} toggles - feature toggles object, based on api response
- * @returns {string[]} - list of paths for unreleased pages
- */
-function getUnreleasedPages(toggles) {
-  // if toxic exposure toggle is disabled, add those pages to the unreleased pages list
-  if (getToggleValue(toggles, 'disability_526_toxic_exposure') !== true) {
-    return Object.keys(toxicExposurePages).map(page => {
-      return toxicExposurePages[page].path;
-    });
-  }
-
-  return [];
-}
-
-/**
- * For each unreleased page, create the page hook to throw an error if the page loads
- * @param {object} toggles - feature toggles object, based on api response
- * @returns {object} object with page hook for each unreleased page
- */
-function makeUnreleasedPageHooks(toggles) {
-  const pages = getUnreleasedPages(toggles);
-
-  return Object.assign(
-    {},
-    ...pages.map(path => {
-      return {
-        [path]: () => {
-          throw new Error(`Unexpectedly showing unreleased page [${path}]`);
-        },
-      };
-    }),
-  );
-}
-
-export const pageHooks = (cy, toggles = mockFeatureToggles) => ({
+export const pageHooks = cy => ({
```

Comment:
The CC team is using a toggle check in a page hook so I don't think we can remove this (see [pr 30152](https://github.com/department-of-veterans-affairs/vets-website/pull/30152/files))

But also, we want to keep the unreleased pages verification for TE. It would be very bad if we had another unreleased page bug. Let's pair on this when you have time. I have some ideas and it shouldn't take much time.